cookbook/core/containerization/spot_instances.py

```diff
@@ -1,8 +1,53 @@
 """
 Using Spot Instances
-----------------------
+--------------------
+
+What Are Spot Instances?
+========================
+
+Spot Instances are unused EC2 capacity in AWS. `Spot instances <https://aws.amazon.com/ec2/spot/?cards.sort-by=item.additionalFields.startDateTime&cards.sort-order=asc>`_ can result in up to 90% savings on on-demand prices. The caveat is that at any point these instances can be preempted and no longer be available for use. This can happen due to:
+
+* Price – The Spot price is greater than your maximum price.
+* Capacity – If there are not enough unused EC2 instances to meet the demand for Spot Instances, Amazon EC2 interrupts Spot Instances. The order in which the instances are interrupted is determined by Amazon EC2.
+* Constraints – If your request includes a constraint such as a launch group or an Availability Zone group, these Spot Instances are terminated as a group when the constraint can no longer be met.
+
+Generally, most spot instances are obtained for around 2 hours (median), with the floor being around 20 minutes, and the ceiling of unbounded duration.
+
+What Is Interruptible?
+======================
+
+Anyone looking to realize cost savings should look into interruptible.
+Interruptible allows users to specify that their tasks are ok to be scheduled on machines that may get preempted such as AWS spot instances. 
```

Comment:
```suggestion
If specified, the interruptible flag is added to the task definition, and signals to the Flyte engine that it may be scheduled on machines that may be preempted, such as AWS spot instances.  This is low-hanging fruit for any cost-savings initiative. 
```